commands/root.go

```diff
@@ -138,9 +139,28 @@ func decorateRootCommand(cmd *cobra.Command) {
 	}
 }
 
+func subCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		clusterCmd.ClusterCmd,
+		mapCmd.MapCmd,
+	}
+	fds := []fakeDoor.FakeDoor{
+		{Name: "list", IssueNum: 42},
+		{Name: "queue", IssueNum: 43},
+		{Name: "multimap", IssueNum: 44},
+		{Name: "replicatedmap", IssueNum: 45},
+		{Name: "set", IssueNum: 46},
+		{Name: "topic", IssueNum: 47},
```

Comment:
I've created the tickets for these mocks. Could you change the issue numbers? From 48 to 53?

Reply:
Changed [fb5d42a](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/fb5d42aba41293e47a54b3b97b1c893b7a0975ea).